static/docs/commands-reference/status.md

```diff
@@ -53,19 +53,40 @@ This says that no differences were detected, and therefore that no stages would
 be run again if `dvc repro` were executed.
 
 If instead, differences are detected, `dvc status` lists those changes. For each
-DVC-file (stage) with differences, the _dependencies_ and/or _outputs_ that
-differ are listed. For each item listed, either the file name or the checksum is
-shown, and additionally a status word is shown describing the change:
+DVC-file (stage) with differences, the changes in both _dependencies_ and/or
```

Comment:
Can't use "both" with "and/or". I'd remove "both" since "and/or" is more correct.